website/content/docs/pipelines/index.mdx

```diff
@@ -0,0 +1,439 @@
+An "Up" was a special operation that encapsulated a Build, Deploy, and
+Release for an application. For most users getting started with Waypoint, this
+simple pipeline is powerful enough to get off the ground continuously building
+and deploying an application to production.
+
+However, as projects and teams
+become more complicated addressing real-world concerns, users wished for more
+control and customization over the order in which Waypoint deploys applications.
+Teams need to run tests after a build, security scan artifacts prior to a deploy, and
+so on. Custom Pipelines enable teams to solve these problems. With this tech preview, users can
+define pipelines to run in the order they require to deliver their
+application. Additionally, pipelines allow for custom execution steps through
+the "Exec" plugin, allowing teams to run scripts as part of the pipeline
+execution, further enabling highly-customized operations.
+
+Pipelines allow users to tell Waypoint what order these operations should
+run in and configure any operation options necessary, like disabling an
+artifact push in the `build` step, or pruning old deployments
+on a release in the `release` step.
+
+Please visit the [Pipeline Configuration Docs](/docs/pipelines/config)
+for more information about the configuration of a Pipeline for a
+Waypoint project.
+
+### Pipeline Runs
+
+Each time a pipeline is run, Waypoint keeps track of the steps and run results.
+As you can see from the [waypoint pipeline run console example output](/docs/pipelines#examples),
+Waypoint identifies how many steps are in the pipeline and how many are successfully
+completed. Users can then further deep dive into specific pipeline runs to view
+more information.
+
+`waypoint pipeline list` shows the current steps, last run start and complete
+times, last run state, and total runs for every pipeline in the project.
+
+```shell
+❯ waypoint pipeline list
+
+» Waypoint Pipelines for my-project
+              ID             | NAME |    OWNER     | CURRENT STEPS | LAST RUN STARTED | LAST RUN COMPLETED |  STATE  | TOTAL RUNS
+-----------------------------+------+--------------+---------------+------------------+--------------------+---------+-------------
+  01GBBBC63TR5DYQXQVGJPS8JGZ | test | example-java |             2 | 5 days ago       | 5 days ago         | success |          5
+```
+
+`waypoint pipeline inspect` can look up a specific pipeline by ID (default) or
+name, as well as an additional option to view a specific run of that pipeline.
+Pipeline run information will include run state, steps, timestamps, and associated job IDs.
+
+```shell
+❯ waypoint pipeline inspect -name=test
+
+
+» Pipeline Configuration
+                  ID: 01GBBBC63TR5DYQXQVGJPS8JGZ
+                Name: test
+               Owner: example-java
+      Root Step Name: build
+         Total Steps: 2
+          Total Runs: 9
+    Last Run Started: 1 minute ago
+  Last Run Completed: a long while ago
```

Comment:
nit: while is recognized as a keyword here and shows up in red text.

<img width="899" alt="Screen Shot 2022-09-07 at 12 28 53 PM" src="https://user-images.githubusercontent.com/83741749/188931213-d8b243a3-c67a-4e33-9189-5e689fd85ef6.png">

Reply:
That's because it's wrapped in a `shell` block, not much we can do about this! Could make it a regular code block with no formatting.